src/platform/src/main/java/org/geoserver/platform/ModuleStatusImpl.java

```diff
@@ -0,0 +1,124 @@
+/* (c) 2016 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.platform;
+
+/**
+ * Bean used to register module installation in applicationContext.xml.
+ * <p>
+ * Bean completly defined by applicationContext.xml - no dynamic content.
+ * <pre>
+ * &lt!-- code example needed --&gt;>
+ * </pre>
+ * @author Morgan Thompson - Boundless
+ */
+public class ModuleStatusImpl implements ModuleStatus {
+
+    private String module;
+    private String component;
+    private String name;
+    private String version;
+    private String documentation;
+    private String message;
+    
+    public ModuleStatusImpl() {
+    }
+    public ModuleStatusImpl(ModuleStatus status){
+        this.module = status.getModule();
+        this.component = status.getComponent();
+        this.name = status.getName();
+        this.version = status.getVersion();
+        this.documentation = status.getDocumentation();
+        this.message = status.getMessage();
+    }
+
+    public ModuleStatusImpl(String description) {
+        String split[] = description.split(",");
```

Comment:
Why are you passing in a comma separated string rather than two separate strings?